user-guide/docs/stylesheets/extra.css

```diff
@@ -1,6 +1,6 @@
 :root {
-  --color-main: #003245;
-  --color-secondary-blue: #2aafc8;
+  --color-main: #191A25;
+  --color-secondary-blue: #007BA7;
   --color-secondary-blue-200: rgb(175, 229, 238);
 
   --color-secondary-400: #008ca4;
```

Comment:
Can you update these 2 variables with the suggestions below to match the navs of main docs site.

  --color-secondary-400: #bb0c12;
  --color-secondary-200: #e51d24;

Reply:
Corrected it. Please take another look.